lib/github/github.ex

```diff
@@ -26,6 +26,19 @@ defmodule BorsNG.GitHub do
   @type tcollaborator :: %{user: tuser, perms: tuser_repo_perms}
   @type tcommitter :: %{name: bitstring, email: bitstring}
 
+  @spec get_pr_files!(tconn, integer | bitstring) :: [BorsNG.GitHub.File.t]
```

Comment:
If I am not mistaken, pr_xref is integer only

Reply:
If it's not, fix it so it is.

Reply:
⚡️